core/src/main/scala/org/apache/spark/internal/config/UI.scala

```diff
@@ -220,8 +220,20 @@ private[spark] object UI {
     .stringConf
     .createOptional
 
+  val CUSTOM_DRIVER_LOG_URL = ConfigBuilder("spark.ui.custom.driver.log.url")
```

Comment:
Since this doesn't have `spark.kubernetes.` prefix, this is supposed to work with YARN, @EnricoMi . Are you sure?

Reply:
This was just derived from `spark.ui.custom.executor.log.url`, where I suspect this should be similar. The property name does not mention Yarn, and documentation should say that resource managers **may** support this.